bionic/decorators.py

```diff
@@ -254,9 +253,7 @@ def outputs(*names):
         A decorator which can be applied to an entity function.
     """
 
-    return decorator_updating_accumulator(
-        lambda acc: acc.wrap_provider(NameSplittingProvider, names)
-    )
+    return returns(",".join(names) + ",")
```

Comment:
Now that we are done with this change, what do you think about marking `returns` decorator stable and deprecating `outputs` in favor of `returns`? Or do you want to wait till we have another descriptor that uses `returns`?

Reply:
Since it doesn't really enable any new functionality compared to `@output`/`@outputs`, I'd like to leave `@returns` undocumented for now.

In fact, I'm leaning towards leaving it undocumented/experimental for a while, and instead continuing to add special-purpose decorators like `@output_as_path`, `@arg_as_path`, `@output_as_future`, which would just be wrappers around the descriptor functionality. That way we can keep iterating on the descriptor syntax, and it'll be easier to document them when they're a coherent architectural feature instead of a small collection of tricks.